### PR TITLE
chore: remove empty styles and make className last in components(EDS)

### PR DESCRIPTION
**Pull request by schakraborty22**
### Summary:
This is PR to implement the removal of empty styles classNames in the components and make the className props last in the conditional styling (cslx) so that the dev styles will override the props styles passed to the components:
Story id -
204928 : https://app.shortcut.com/czi-edu/story/204928/make-classname-prop-the-last-class-in-list-for-all-components
211218: https://app.shortcut.com/czi-edu/story/211218/remove-classnames-that-don-t-have-corresponding-styles

### CODE REVIEW

File: src/components/StackedBlock/StackedBlock.tsx [hunk 1]
```diff
@@ -41,10 +40,8 @@ export const StackedBlock = ({
   children,
   ...other
 }: Props) => {
-  const componentClassName = clsx(styles['stacked-block'], className);
```
Comment (dierat) [inline src/components/StackedBlock/StackedBlock.tsx:44]: Could you also remove that [empty style block from the css modules file](https://github.com/chanzuckerberg/edu-design-system/blob/998b6168aa3ad73d81606d0efd3236da1fd6f154/src/upcoming-components/StackedBlock/StackedBlock.module.css#L12)?

File: src/components/Tab/Tab.module.css [hunk 2]
```diff
@@ -1,5 +0,0 @@
-@import '../../design-tokens/mixins.css';
```
Comment (dierat) [inline src/components/Tab/Tab.module.css:1]: Nice!

File: src/components/TableObjectHeader/TableObjectHeader.tsx [hunk 3]
```diff
@@ -23,11 +23,7 @@ export interface Props {
  * TODO: update this comment with a description of the component.
  */
 export const TableObjectHeader = ({ children, className, ...other }: Props) => {
-  const componentClassName = clsx(
-    styles['table-object__header'],
-    className,
-    {},
```
Comment (dierat) [inline src/components/TableObjectHeader/TableObjectHeader.tsx:29]: Thanks for removing all these empty objects too ❤️

File: src/components/StackedBlock/StackedBlock.tsx (hunk 1 above)
Comment (schakraborty22) [reply src/components/StackedBlock/StackedBlock.tsx:44]: @dierat, thats a good idea, will remove thanks!

**Review by dierat — APPROVED** [review]
This is great, thanks! ❤️

**Pull request merged**